package.json

```diff
@@ -14,6 +14,7 @@
     "d3-selection": "3.0.0",
     "d3-zoom": "3.0.0",
     "font-awesome": "4.7.0",
+    "highlight.js": "^11.10.0",
```

Comment:
chore: Pin the version of the new dependency to stay on the current versioning pattern for this repo
```suggestion
    "highlight.js": "11.10.0",
```

Reply:
Done https://github.com/HathorNetwork/hathor-explorer/pull/293/commits/40aba4857a7fec3680eddd1f6365dee1c2766d82